### PR TITLE
Add Docs for the WP-DS-NPR-API Plugin

### PR DESCRIPTION
**Pull request by jackbrighton**
## What was done

Added:
- a readme.txt file to the repo and validated it against the WordPress readme validator
- (https://wordpress.org/plugins/about/validator/)
- Fleshed out readme.md
- Added folder for docs for docs
- Added folder for assets for docs images
- Added docs for installation, settings, pulling single stories, pulling multiple stories, pushing stories to the API, pushing to NPR One, and Field Mapping
## Why?

Docs did not exist in the plugin repo. Creating them was part of our current contract with NPR DS. See project task at https://app.asana.com/0/100231216298059/130990810919562
## What Next:

This Docs branch should be merged with INN/WP-DS-NPR-API master so when Ryan's improvements to the plugin code are complete the whole thing can be pushed to NPR/WP-DS-NPR-API and the WordPress plugin directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,175 @@
...
+
+== Description ==
+
+WordPress NPR Story API Plugin provides push and pull functionality with the NPR API along with a user-friendly administrative interface.
+
+NPR's API is a content API, which essentially provides a structured way for other computer applications to get NPR stories in a predictable, flexible and powerful way. The content that is available includes audio from most NPR programs dating back to 1995 as well as text, images and other web-only content from NPR and NPR member stations. This archive consists of over 250,000 stories that are grouped into more than 5,000 different aggregations.
+
+Access to the NPR API requires an API Key which you can get by [registering for an NPR account](http://www.npr.org/account/signup).
+
+The WordPress plugin is being developed as an Open Source plugin by NPR Digital Services. If you would like to suggest features or bug fixes, or better yet if you would like to contribute new features or bug fixes please contact Digital Services through our support page at [http://info.ds.npr.org/support.html](http://info.ds.npr.org/support.html).
```
Comment (aschweigert) [inline README.md:25]: it's ok to mention this here, but the repo should also have a contributing.md file at the root level that covers how to contribute in more detail

File: README.md [hunk 2]
```diff
@@ -1,2 +1,175 @@
...
+
+![Getting NPR Stories by Story ID](/assets/get-npr-stories-link.png)
+
+NPR Stories having got gotten
+
+![NPR Stories having got gotten](/assets/npr-stories.png)
+
+
+== Changelog ==
+
```
Comment (aschweigert) [inline README.md:78]: is our update 1.6? if so, changes should be captured here before we tag the new release

**Comment by aschweigert** [comment]
for clarity's sake, let's put the screenshots in /docs/assets/img instead of directly in /assets to make it clear that these are related only to docs and not used by the plugin itself

**Comment by jackbrighton** [comment]
OK the docs images are now in docs/assets/img/ and there's also now a contributing.md file in the plugin root directory.

**Pull request merged**